### PR TITLE
new feature: dlbcl90 genes

### LINKED ISSUE #98: Feature request: DLBCL90 gene lists
Would it be straightforward to integrate the gene lists found here into this repo? 

`/projects/dscott_prj/CCSRI_1500/shared/ref/DLBCL90`

### PR DESCRIPTION
**Pull request by Kdreval**
This is to address #98

### CODE REVIEW

**Comment by Kdreval** [comment]
Also took this as opportunity to add the new logo to the README 😄

File: GAMBLR.data.png
Comment (lkhilton) [inline GAMBLR.data.png:1]: 🚀

File: R/data.R [hunk 1]
```diff
@@ -1010,3 +1010,19 @@
...
 "cytobands_hg38"
+
+#' DLBCL90 genes
+#'
+#' A data frame with genes and their weights for DLBCL90.
+#'
+#' @format ## `dlbcl90_genes`
+#' A data frame with 73 rows and 6 columns
+#' \describe{
+#'   \item{Gene.symbol}{Human-readable gene symbol.}
```
Comment (lkhilton) [inline R/data.R]: Does it make sense to add a comment as to why there are two nearly identical gene symbol columns? Maybe: 

```
{Gene.symbol}{Human-readable gene symbol matching the symbols used for the DLBCL90 code set design.} 
...
{hgnc_symbol}{Human-readable gene symbol matching Gencode 33.}
```

Comment (Kdreval) [reply R/data.R]: Thanks! Added this and pushed the changes! ✅

**Review by lkhilton — APPROVED** [review]

**Pull request merged**